petisco/extra/rabbitmq/application_rabbitmq_message_configurer.py

```diff
@@ -0,0 +1,35 @@
+import os
+from distutils.util import strtobool
+from typing import List, NoReturn
+
+from petisco.base.application.application_configurer import ApplicationConfigurer
+from petisco.base.application.dependency_injection.container import Container
+from petisco.base.domain.message.message_subscriber import MessageSubscriber
+
+MAX_RETRIES = 5
+
+CLEAR_SUBSCRIBER_BEFORE = strtobool(
+    os.getenv("PETISCO_RABBITMQ_CONFIGURER_CLEAR_SUBSCRIBER_BEFORE", "false")
+)
+CLEAR_STORE_BEFORE = strtobool(
+    os.getenv("PETISCO_RABBITMQ_CONFIGURER_CLEAR_STORE_BEFORE", "false")
+)
+
+
+class ApplicationRabbitMqMessageConfigurer(ApplicationConfigurer):
+    def __init__(
+        self,
+        execute_after_dependencies: bool = False,
```

Comment:
Since `execute` need a dependency (`Container.get("message_configurer")`) would better set default value of `execute_after_dependencies` to `True`?

Reply:
Alright!!! It works for me!